zmon_aws_agent/aws.py

```diff
@@ -360,10 +360,13 @@ def get_running_elbs_application(region, acc):
             'elb_type': 'application',
             'dns_name': e['DNSName'],
             'host': e['DNSName'],
-            'name': name,
+            'name': '/'.join(arn.rsplit('/')[-3:]),  # name used by Cloudwatch!
```

Comment:
I noticed that Cloudwatch checks use `entity['name']` so this is why I adjusted the name here, while keeping the original as `elb_name`.